css/index.css

```diff
@@ -0,0 +1,168 @@
+.side-navbar {
+  background-color: rgb(25 146 212);
+  width: 200px;
+}
+
+.menuitem {
+  display: flex;
+}
+
+img {
```

Comment:
@Gbemisola19 Nice work so far, but why don't we give this a class name and style the class name instead, it helps simplify the code and also reusable

Reply:
I gave it a class name already, or can you expatiate more

Reply:
@Gbemisola19 I should have added a documentation that explains more about that,
[HTML Class Name](https://www.w3schools.com/html/html_classes.asp)
Take a look at that